components/browser/state/src/main/java/mozilla/components/browser/state/state/ContentState.kt

```diff
@@ -46,6 +46,7 @@ import mozilla.components.concept.engine.window.WindowRequest
  * @property loadRequest last [LoadRequestState] if this session.
  * @property permissionRequestsList Holds unprocessed content requests.
  * @property appPermissionRequestsList Holds unprocessed app requests.
+ * @property refreshCanceled Indicates if an intent of refreshing was canceled.
```

Comment:
nit: True if a page refresh was cancelled by the user. Note that this is not about stopping an ongoing page load but useful in cases like swipe-to-refresh which allow users to cancel or abort before a page is refreshed.

Reply:
Let's make this as clear as possible ^^.

Reply:
👍🏽